msg/en_US.utf8/utils.msg

```diff
@@ -1330,4 +1330,6 @@ valid options:\n\
                                   WARNING: This option has to be used with '-t(--transaction)' option.\n\
     -m, --module=TYPE             display detailed memory usage information of the specified TYPE.\n\
                                   TYPE of modules\n\
-                                  -1      all modules\n
+                                  -1      all modules\n\
+                                   0      heap module\n\
+                                   1      common module\n
```

Comment:
I'd put `common module` before `heap module`. If other modules come in, it will looks strange. It'd make `heap module` special.

Reply:
Done.